### ISSUE TITLE
Hypnotix 4.3 - Xtream API : AttributeError: 'Channel' object has no attribute 'info'

### ISSUE THREAD

**Issue by mrwebfr** [linuxmint/hypnotix#326]
Hello,
Hypnotix 4.3 bug when launching a TV channel with Xtream API on Ubuntu 23.10.
Installed by http://packages.linuxmint.com/search.php?release=any&section=any&keyword=hypnotix

```
AttributeError: 'Channel' object has no attribute 'info'
CHANNEL: '|FR| ........' (http://..............ts)
Traceback (most recent call last):
  File "/usr/lib/hypnotix/hypnotix.py", line 932, in before_play
    data = f"{channel.info}:::{channel.url}"
```
**Solution**

I fixed the problem by deleting ```{channel.info}```.
```data = f"{channel.info}:::{channel.url}"```
becomes
```data = f"{channel.url}"```

**Comment by superolmo** [comment]
That's right. It is a bug. The channel class does not have the `info` property. It is fixed in the Pull Request #276 
If you know how to, feel free to download PR #276 to test it with your xtream account.
